Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+#include "Person.h"
+
+Person::Person(){
+	age_ = new int();
```

Comment:
I'm not sure if this is part of the generally accepted C style guide, but you can use **initialization lists**.

Ex.
`Person::Person(): age_(new int){}`